core/src/main/java/org/apache/gravitino/proto/RoleEntitySerDe.java

```diff
@@ -21,11 +21,9 @@
 import com.google.common.collect.Lists;
 import java.util.List;
 import java.util.stream.Collectors;
+import org.apache.gravitino.MetadataObject;
 import org.apache.gravitino.Namespace;
-import org.apache.gravitino.authorization.Privilege;
-import org.apache.gravitino.authorization.Privileges;
-import org.apache.gravitino.authorization.SecurableObject;
-import org.apache.gravitino.authorization.SecurableObjects;
+import org.apache.gravitino.authorization.*;
```

Comment:
We don't allow to use wildcard import, can you please fix it?

Reply:
@jerryshao I've fixed the wildcard. I've made the modifications to the code in the comant below.